src/Core/src/Platform/Android/EntryExtensions.cs

```diff
@@ -59,5 +59,13 @@ public static void UpdateIsTextPredictionEnabled(this EditText editText, IEntry
 		{
 			editText.SetInputType(entry);
 		}
+
+		public static void UpdateIsReadOnly(this EditText editText, IEntry entry)
+		{
+			bool isReadOnly = !entry.IsReadOnly;
```

Comment:
Maybe call this `isEditable` instead to avoid confusion.

Reply:
It will be fixed with the next commit :) thanks!